tools/smart_locator_finder.py

```
@@ -474,6 +474,360 @@ async def _find_element_by_description(page, description: str) -> Optional[str]:
+        # Content area patterns (table displaying results)
+        'table displaying', 'displaying results', 'content area of the table',
+        'table content', 'table results', 'results in table'
+    ]
+    
+    # Check if description mentions table rows
+    is_table_row_request = any(keyword in desc_lower for keyword in table_row_keywords)
+    
+    if not is_table_row_request:
+        return None
+    
+    logger.info(f"🔍 TABLE-ROW-FINDER: Description mentions table rows")
```

Ruff (0.14.8)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)